lib/money/money/formatting.rb

```diff
@@ -345,7 +345,7 @@ def regexp_format(formatted, rules, decimal_mark, symbol_value)
 
   def localize_formatting_rules(rules)
     if currency.iso_code == "JPY" && I18n.locale == :ja
-      rules[:symbol] = "円" unless rules[:symbol] == false
+      rules[:symbol] = "円" if rules[:symbol] == true
```

Comment:
Normally I would say just `if rules[:symbol]` instead of explicitly testing against `true`, but I don't know if that's semantically correct for the possible values of `rules[:symbol]` here... maybe @jkingdon or @semmons99 has an opinion